libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -49,6 +49,10 @@ extern AP_IOMCU iomcu;
 #endif
 
 #define RCOU_SERIAL_TIMING_DEBUG 0
+#define LED_THD_WA_SIZE 256
+#if !defined(HAL_NO_LED_THREAD) && defined(HAL_NO_RCOUT_THREAD)
+#define HAL_NO_LED_THREAD
```

Comment:
```suggestion
#define HAL_NO_LED_THREAD 1
```

... also, make sure this is always defined and stop checking for definition elsewhere, check for boolean value.  This prevents bugs where we have different include orders leading to different values for this define